### PR TITLE
Support multiple library versions per Gradle subproject

### LINKED ISSUE #49: Support multiple versions for the same artifact
As mentioned in #41, further actions should include allowing for multiple versions of processed library within one Gradle project. Some details can be found in linked issue, with more detailed description TBD.

### PR DESCRIPTION
**Pull request by r4zzz4k**
I've tried to cover details in docs. Of course one can refer actual projects to see API. Right now we've got material covered.

Closes #49.

### CODE REVIEW

File: anvil-appcompat-v7/src/gen-appcompatV7-1.1.0/kotlin/dev/inkremental/dsl/androidx/appcompat/view/menu/ActionMenuItemView.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ import kotlin.CharSequence
 import kotlin.Suppress
 import kotlin.Unit
 
-fun actionMenuItemView(configure: ActionMenuItemViewScope.() -> Unit = {}) =
+fun ActionMenuItemView(configure: ActionMenuItemViewScope.() -> Unit = {}) =
```
Comment (sgrekov) [inline anvil-appcompat-v7/src/gen-appcompatV7-1.1.0/kotlin/dev/inkremental/dsl/androidx/appcompat/view/menu/ActionMenuItemView.kt]: What are the reasons that we are changing dsl calls to uppercase?

**Review by sgrekov — CHANGES_REQUESTED** [review]

File: anvil-appcompat-v7/src/gen-appcompatV7-1.1.0/kotlin/dev/inkremental/dsl/androidx/appcompat/view/menu/ActionMenuItemView.kt (hunk 1 above)
Comment (r4zzz4k) [reply anvil-appcompat-v7/src/gen-appcompatV7-1.1.0/kotlin/dev/inkremental/dsl/androidx/appcompat/view/menu/ActionMenuItemView.kt]: Seems to be an error, thanks for catching.

File: build.gradle.kts [hunk 2]
```diff
@@ -39,15 +39,16 @@ subprojects {
...
 val mainSubprojects = listOf(
     "anvil",
     "anvil-appcompat-v7",
     "anvil-gridlayout-v7",
     "anvil-recyclerview-v7",
     "anvil-cardview-v7",
     "anvil-design",
-    "anvil-support-v4"
+    "anvil-support-v4",
+    "anvil-constraintlayout",
```
Comment (sgrekov) [inline build.gradle.kts:50]: 👍

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -24,6 +24,105 @@ sdkmanager \
     "tools"
 ```
 
+# Adding new libraries
```
Comment (sgrekov) [inline docs/CONTRIBUTING.md:27]: Wow, great job describing it! I have good candidates for "playgrounding" with this guide - Fresco and Lottie.

**Review by sgrekov — APPROVED** [review]

**Review by Mishkun — APPROVED** [review]

**Pull request merged**